tests/gui/test_DirectWaitBar.py

```diff
@@ -0,0 +1,9 @@
+# coding=utf-8
+from direct.gui.DirectWaitBar import DirectWaitBar
+
+def test_create_waitbar():
+    bar = DirectWaitBar(pos = (0, 0, 0), scale = (0.1, 0.1, 0.1))
```

Comment:
There should not be spacing around `=` in keyword argument specifications in function call (don't ask me why)